app/js/components/layouts/Tags.react.js

```diff
@@ -1,28 +1,28 @@
 import React from 'react'
+import _ from 'jquery'
 
-const Tag = (tag) => {
-     return (
-       <span>
-         {tag}
-       </span>
-     )
- }
+class Tag extends React.Component{
```

Comment:
both this and the components should be written in the function based component. since there isn't another dependent method.

What I mean is this could have been written as
```javascript 
const Tag = ({props}) => {
return (<span> {props.tag </span)
}
```

const  Tags = ({props}) => {
const { tags } = this.props;  
return ( .. <Tag key={i} tag={tagsi[i]}>)
}```